quiz.rb

```diff
@@ -2,16 +2,47 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+class Dog
+ def initialize(name,breed,age)
+  @name = name
+  @breed = breed
+  @age = age
+     attr_reader :name
+     attr_reader :breed
+     attr_reader :age
+   end
+
```

Comment:
Close! A couple of small things:
- It looks like you forgot to close your `initialize` method with an `end`; the way it's written now, your `attr_reader`s are inside the initialize method instead of outside. Remember, just like with JavaScript, any time you have matching pairs (be it `class ... end` or `def ... end`) it's a good idea to create them at the same time so that you don't forget later.
- Rather than listing them all separately, you can actually put all your `attr_reader` attributes into a single line, like this:
`attr_reader :name, :breed, :age`